src/frontend/src/app/app.module.ts

```diff
@@ -64,6 +64,8 @@ import { EffectsModule } from '@ngrx/effects';
 import { reducers, metaReducers } from './reducers';
 import { ResourceDefinition, NgrxJsonApiModule } from 'ngrx-json-api';
 import { UsersComponent } from './components/users/users.component';
+import { FormsModule } from '@angular/forms';    // add this
+import { UserService } from './services/user.service';    // add this
```

Comment:
We may want to remove the comments to avoid confusion later down the road.

Reply:
Sure thing.